Automattic/wp-calypso
# IAP: Disable all links & buttons that add upgrade to the cart

It partially fixes #291.

Features covered in this PR:
* [x] Domains search
* [x] Renew button
* [x] Credit card list and edit

### Testing
1. Disable the following flags in the config file:
 * `upgrades/checkout`
 * `upgrades/credit-cards`
 * `upgrades/domain-search`
2. Execute `make run` and go to http://calypso.localhost:3000/purchases.
3. Select a Purchase that expires soon.
4. On Manage Purchase page there should be no Renew button.
5. You shouldn't be able to edit Credit Card details.
6. Go to Billing History (http://calypso.localhost:3000/me/billing) and scroll to the bottom of the page.
7. You shouldn't see list of credit cards.
8. Go to Domain (http://calypso.localhost:3000/domains/manage).
9. Select a Site.
10. You shouldn't see any button that allows to add domain.
11. Go to Site Settings (http://calypso.localhost:3000/settings/general).
12. Select a Site.
13. In Site Address section you shouldn't see any button or link which allows to add domain, domain mapping or site redirect.
14. There should be no information about domain registration in Change Site Address.

Now enable flags listed in (1) and repeat all other steps, you should see all action buttons and links :)
